src/app/pages/home/profile/profile.page.ts

```diff
@@ -30,6 +30,107 @@ export class ProfilePage implements OnInit {
     targetHeight: 374
   }
 
+  slideOpts = {
```

Comment:
I'll run it locally in the morning but I don't see how any of this is related.

Reply:
Related to...? The issue is related to updating the view of each page.